### DISCUSSION TITLE
Ability to add custom icon per Jackett provider

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#8124] [Ideas & Feature requests]
As a workaround to https://github.com/pymedusa/Medusa/issues/8119 I've added the tracker as a Jackett provider which works perfect.
The downside is that when you do a manual search you will only see the Jackett icon on the results, when you have multiple Jackett providers that makes it harder to see which you want to download.

(At the moment I only have 1 Jackett provider)
![image](https://user-images.githubusercontent.com/4103090/82727451-fbdbfc80-9cea-11ea-9a2c-d6e7cf35e314.png)

It would be really helpful if you could add a custom icon to the Jackett Provider.
![image](https://user-images.githubusercontent.com/4103090/82727507-30e84f00-9ceb-11ea-891a-1e34e26c8334.png)

**Comment by medariox** [comment]
FYI this is already possible by adding an image called like the provider to `assets/img/providers` e.g.: `assets/img/providers/blutopia.png`. If you submit a PR we can add it to the next release.

**Comment by Rouzax** [comment]
So if there is an icon with the same name as the one I created as Jackett provider, that one will be picked up?

**Comment by medariox** [comment]
Yes 😏

**Comment by Rouzax** [comment]
Just tested on my local install and that does indeed work 👍 

I've never created a pull request, let me see how that works 😄 
Attached is the icon, just in case:
![blutopia](https://user-images.githubusercontent.com/4103090/83412849-20d31c80-a41b-11ea-95aa-3766ac318e54.png)

EDIT:
Think I managed to do it, hopefully it is okay.
https://github.com/pymedusa/Medusa/pull/8163
